docs/template.asciidoc

```diff
@@ -0,0 +1,62 @@
+[[page-title]]
+= Page title
```

Comment:
We need some guidelines on page titles. 

Put important words first. Make it easy to scan in the TOC.  Not everything needs to start with Managing